runtime/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -140,29 +93,10 @@ public constructor(
     }
 
     /**
-     * Deserializes [json] element into a corresponding object of type [T] using provided [deserializer].
-     * @throws [JsonException] in case of malformed json
-     * @throws [SerializationException] if given input can not be deserialized
+     * The default instance of [Json] with default configuration.
      */
-    public fun <T> decodeFromJsonElement(deserializer: DeserializationStrategy<T>, json: JsonElement): T {
-        return readJson(json, deserializer)
-    }
-
-    /**
-     * Deserializes [element] element into a corresponding object of type [T] using serializer registered in the module.
-     * @throws [JsonException] in case of malformed json
-     * @throws [SerializationException] if given input can not be deserialized
-     */
-    public inline fun <reified T : Any> decodeFromJsonElement(element: JsonElement): T =
-        decodeFromJsonElement(context.getContextualOrDefault(), element)
-
-    /**
-     * The default instance of [Json] in the form of companion object. Configured with [JsonConfiguration.Default].
-     */
-    @UnstableDefault
-    public companion object Default : StringFormat {
-
-        private val jsonInstance = Json(JsonConfiguration.Default)
+    public companion object Default : Json(JsonConfiguration.Default) {
+        private val jsonInstance = Json(JsonConfiguration.Default, EmptyModule)
```

Comment:
Why do we have to use delegation here? I think we can move all implementation from JsonImpl to Json, because the only difference between Default and JsonImpl is configuration, which can be passed to `Json()` constructor.

Reply:
Good point, thanks for spotting